### ISSUE TITLE
Possibly unwanted apps on a stock installation without gapps

### ISSUE THREAD

**Issue by pitsi** [crdroidandroid/issue_tracker#194]
### Issue type

Other

### Device

Galaxy A40 (a40)

### crDroid version

crDroid 8

### Exact version / Build date

8.2/2022-02-13

### Bug description

As seen on the screenshot from aurora store below, the app wants to update 2 apps that do not seem to be installed... at first sight. However, their package names appear on the list of installed packages here, so this means they came preinstalled with the rom...

https://paste.debian.net/hidden/8abe52dd/

Is their inclusion a dev preference or am I missing something?

### Steps to reproduce

1) Install the rom with the regular procedure (twrp, wiping, etc)
2) Install aurora store from either its apk or via fdroid.
3) Launch aurora store and go to the updates page.

### Relevant log (logcat/build log)

```shell
(not needed)
```

### Screenshots or videos

![243087031-d7a7133a-edfd-4adc-a118-e672c3fc8d58](https://github.com/crdroidandroid/issue_tracker/assets/5880037/2971d487-5f58-4014-a343-25630b92e48f)

### Solution

No idea.

### Additional context

Tbh, since this is the first time I install crdroid on a device, I expected it to be completely free of google's apps, and that includes the device health services. 
Likewise, I expected it to have an opensource camera app and not a closed source one that, according to play store's description, has ads and in-app purchases. That last one is the reason I do not want to update the app, because it may get ads.

p.s. The device is not rooted, so I can't "just uninstall it".

### Acknowledgements

- [X] I've checked device is officially supported (for device specific reports and not source related).
- [X] I'm running latest version available on crdroid.net for this device
- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by pitsi** [comment]
Thank you for the clarifications. If I update footej, will it remain ad free?

**Comment by gwolf2u** [comment]
To be honest am not sure

**Comment by pitsi** [comment]
Oh well, I will take my chances then :D

**Comment by pitsi** [comment]
Double failure :D
The footej camera app can not be updated due to "not matching signatures" and it just says "conflicting package".
The google device health services can not be updated because it can not be downloaded... and it can not be downloaded because the app "was not bought"!

**Comment by pitsi** [comment]
For the record, I reflashed the rom and this time I installed some gapps package which provides only the store (plus anything else it needs to function).
The store did update the device health services package, but the other camera package is not listed as an update in there.

Sidenote. The camera app was removed (?) from the store a few days ago, but it was added ack (?). I could not find it even from the store's web page on a desktop browser.
